Sources/Persistence/DiskMemoryPersistenceStack.swift

```diff
@@ -249,7 +248,7 @@ public extension Persistence {
                     try self.remove(fileAtURL: fileURL, size: fileSize)
 
                 } catch let error as NSError where error.isNoSuchFileError {
```

Comment:
`isNoSuchFileError` is true on `NSFileReadNoSuchFileError` but there is a similar error `NSFileNoSuchFileError` and I can't tell if `NSFileManager.removeItem` might also throw it or not.

Reply:
Nice catch on `NSFileNoSuchFileError`, will handle it 👍 (even though I haven't seen it in the wild occurring from the `URL.resourceValues` call (which performs a file read)).

Regarding the `NSFileManager.removeItem` call (made inside `self.remove()`), at that point we're assuming the file should exist, and we wrap any error in a `Error.failedToRemoveFile(.removeFailed(error))` which is propagated upstream.